ros2_sa_build.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -xe
+
+export SCRIPT_DIR=$(dirname ${DOCKER_BUILD_SCRIPT})
+
+# install dependencies
+apt-get update && apt-get install -q -y dirmngr curl gnupg2 lsb-release zip python3-pip python3-apt dpkg
+pip3 install -U setuptools
+
+curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
+sh -c 'echo "deb [arch=amd64,arm64] http://packages.ros.org/ros2/ubuntu `lsb_release -cs` main" > /etc/apt/sources.list.d/ros2-latest.list'
+apt-get update && apt-get install --no-install-recommends -y python3-rosdep python3-rosinstall python3-colcon-common-extensions ros-$ROS_DISTRO-ros-base
+pip3 install colcon-bundle colcon-ros-bundle
+
+# Get latest colcon bundle
+COLCON_BUNDLE_INSTALL_PATH="${HOME}/colcon-bundle"
+rm -rf "${COLCON_BUNDLE_INSTALL_PATH}"
+COLCON_ROS_BUNDLE_INSTALL_PATH="${HOME}/colcon-ros-bundle"
+rm -rf "${COLCON_ROS_BUNDLE_INSTALL_PATH}"
+git clone -b ros2-support https://github.com/colcon/colcon-bundle "${COLCON_BUNDLE_INSTALL_PATH}"
+git clone -b ros2-support https://github.com/colcon/colcon-ros-bundle "${COLCON_ROS_BUNDLE_INSTALL_PATH}"
+
+pip3 install --upgrade pip
+pip install -U --editable "${COLCON_BUNDLE_INSTALL_PATH}"
+pip install -U --editable "${COLCON_ROS_BUNDLE_INSTALL_PATH}"
+
+# Remove the old rosdep sources.list
+rm -rf /etc/ros/rosdep/sources.list.d/*
+rosdep init && rosdep update
+
+. /opt/ros/$ROS_DISTRO/setup.sh
+
+BUILD_DIR_NAME=`basename $TRAVIS_BUILD_DIR`
+
+if [ -z "$WORKSPACES" ]; then
+  WORKSPACES="robot_ws simulation_ws"
```

Comment:
I'd like to add support for building the new `log_based_simulation_ws`. could we add that here?

Reply:
I think you could add this by specifying `WORKSPACES` in the .travis.yml file for that project and adding a `WS_BUILD_SCRIPT` for `log_based_simulation_ws`